org.kde.kanagram.json

```diff
@@ -0,0 +1,29 @@
+{
+    "id": "org.kde.kanagram",
+    "branch": "master",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15",
+    "sdk": "org.kde.Sdk",
+    "command": "kanagram",
+    "tags": ["nightly"],
+    "desktop-file-name-suffix": " (Nightly)",
+    "finish-args": ["--share=network", "--socket=pulseaudio", "--share=ipc", "--socket=x11", "--socket=wayland" ],
+
+    "modules": [
+        {
+            "name": "libkeduvocdocument",
+            "buildsystem": "cmake-ninja",
+            "sources": [ { "type": "git", "url": "https://anongit.kde.org/libkeduvocdocument.git" } ]
```

Comment:
We build from stable tarbllas, not from git, please fix this. See https://github.com/flathub/org.kde.kgeography/blob/master/org.kde.kgeography.json for example